canonicalwebteam/discourse/parsers/docs.py

```diff
@@ -465,3 +496,29 @@ def _generate_navigation(self, navigations, version_path):
         )
 
         return navigation
+
+    def _parse_tutorials(self, soup):
+        """
+        Get a list of tutorials topic IDs from all the
+        tutorial tables in a topic
+
+        Example of a tutorial table:
+        | Tutorial |
```

Comment:
```suggestion
        | Tutorials |
```
Would go for plural.

Reply:
Ok, I had doubts about singular or plural. But the table could have more than one so makes sense.